src/wwwroot/app/services/loggerService.js

```diff
@@ -0,0 +1,31 @@
+const OFF = 0;
+const DEBUG_COEFFICIENT = 1;
+const LOG_COEFFICIENT = 2;
+const WARNING_COEFFICIENT = 3;
+const ERROR_COEFFICIENT = 4;
+var logLevel = WARNING_COEFFICIENT;
+
+export default class LoggerService {
+   constructor($log) {
+      'ngInject';
+      this.log = $log;
+   }
+
+   debug() {
+      this.logger(this.log.debug, DEBUG_COEFFICIENT, arguments);
+   }
+   logg() {
```

Comment:
I cannot rename it to log, because the method log() is perceived as a property of $ http service. I will rename it to information() method